pkgs/data/fonts/openmoji/default.nix

```diff
@@ -14,6 +17,34 @@ let
     [ "OpenMoji-Color.ttf" "OpenMoji-Black.ttf" ]
     variant;
 
+  # With newer fontforge the build hangs, see
+  # https://github.com/NixOS/nixpkgs/issues/167869
+  # Patches etc taken from
+  # https://github.com/NixOS/nixpkgs/commit/69da642a5a9bb433138ba1b13c8d56fb5bb6ec05
+  fontforge-20201107 = fontforge.overrideAttrs (old: rec {
+    version = "20201107";
+    src = fetchFromGitHub {
+      owner = "fontforge";
+      repo = "fontforge";
+      rev = version;
+      sha256 = "sha256-Rl/5lbXaPgIndANaD0IakaDus6T53FjiBb45FIuGrvc=";
+    };
+    patches = [
+      (fetchpatch {
+        url = "https://salsa.debian.org/fonts-team/fontforge/raw/76bffe6ccf8ab20a0c81476a80a87ad245e2fd1c/debian/patches/0001-add-extra-cmake-install-rules.patch";
+        sha256 = "u3D9od2xLECNEHhZ+8dkuv9818tPkdP6y/Tvd9CADJg=";
+      })
+      (fetchpatch {
+        url = "https://github.com/fontforge/fontforge/commit/69e263b2aff29ad22f97f13935cfa97a1eabf207.patch";
+        sha256 = "06yyf90605aq6ppfiz83mqkdmnaq5418axp9jgsjyjq78b00xb29";
+      })
+    ];
+    buildInputs = old.buildInputs ++ [ libuninameslist ];
+  });
+  scfbuild-with-pinned-fontforge = scfbuild.override (old: {
```

Comment:
Nit: It seems weird to call one the date and one just "pinned". Maybe just call both `-pinned`?

Reply:
I prefer using the version then. Fixed.